.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       - name: Add SHORT_SHA env property with commit short sha
         run: |
-          SHORT_SHA=$(git rev-parse --short=9 ${{ github.event.pull_request.head.sha }})
+          SHORT_SHA=$(git rev-parse --short=9 HEAD)
```

Comment:
Just confirming; This is the HEAD of the selected branch right like `test-gh-actions` or `master`?

Reply:
It will be HEAD of selected branch.

Reply:
Example: https://github.com/stellar/stellar-etl/actions/runs/10516172476/job/29137804414